bin/local-env/install-wordpress.sh

```diff
@@ -160,6 +160,9 @@ wp plugin install classic-widgets --force --quiet
 echo -e $(status_message "Activating Twenty Twenty theme...")
 wp theme install twentytwenty --activate --force --quiet
 
+echo -e $(status_message "Disable block directory and remote patterns...")
+wp plugin activate e2e-tests-disable-block-directory --quiet
```

Comment:
Instead of this plugin, I wonder if we could use a plugin like [airplane-mode](https://github.com/norcross/airplane-mode).

Reply:
I thought about it as well but it doesn't appear to handle this specific functionality. But it does way more things, so I'd be worried about anything else breaking because of it.

Reply:
Isn't that a good thing, it doesnt call to the internet, might make the tests faster.

Reply:
Does it make sense to activate the plugin here. 

https://github.com/GoogleForCreators/web-stories-wp/blob/f2e523ef807e028475b9467b2441fe0069cd67df/packages/e2e-tests/src/config/bootstrap.js#L251

Reply:
> Does it make sense to activate the plugin here.

`bootstrap.js` is useful for doing things between runs, like deleting media and trashing posts. But in my eyes this is more a once-and-forget-it thing for the environment. Not something that you deactivate later. Just like we activate the RTL Tester once and keep it active.

> Isn't that a good thing, it doesnt call to the internet, might make the tests faster.

Sure, but it would be something _in addition_ to this plugin here. We can give it a try.